### ISSUE TITLE
Second-order constraint constants

### ISSUE THREAD

**Issue by katiemlewis** [switchablenorms/DeepFashion_Try_On#87]
Hi, could you explain where the 0.02 and 0.08 come from in the STN loss calculation?

rg_loss = torch.max(rg_loss,torch.tensor(0.02).cuda())
cg_loss = torch.max(cg_loss,torch.tensor(0.02).cuda())
rx,ry,cx,cy=torch.tensor(0.08).cuda(),torch.tensor(0.08).cuda()\
    ,torch.tensor(0.08).cuda(),torch.tensor(0.08).cuda()

**Comment by LZQhardworker** [comment]
This is a truncation parameter to allow some flexibility. These are given manually. You can try other parameters
